### ISSUE TITLE
Unhandled Promise Rejection

### ISSUE THREAD

**Issue by rafaelfernandezgis** [CartoDB/carto-vl#1131]
Hi there,
I'm getting this error:
```
Unhandled Promise Rejection: DataCloneError: The object can not be cloned.
```
When I try to load [https://carto.com/developers/carto-vl/examples/#example-select-layer](https://carto.com/developers/carto-vl/examples/#example-select-layer).
![captura de pantalla de 2018-11-07 14-14-07](https://user-images.githubusercontent.com/6051197/48133541-6f4c2380-e297-11e8-8d8f-4cdb46ff94c2.png)

It happens on Safari 12.0.1 (14606.2.104.1.1) and Gnome Web (based on WebKitGTK+ 2.22.2)
However it works fine on Chrome, Edge, Firefox and Opera.

It happens no only in your example, but anytime I implement a map using Carto-vl in those browsers I mentioned earlier.

**Comment by rochoa** [comment]
Hey, @rafaelfernandezgis.

We just released a new version 1.0.0, the bug should be solved as per https://github.com/CartoDB/carto-vl/pull/1136. Could you check that version fixes your problem?

Thanks.

**Comment by rafaelfernandezgis** [comment]
It does indeed. Thx @rochoa
